async-nats/src/jetstream/context.rs

```diff
@@ -168,6 +222,29 @@ impl Context {
         }
     }
 
+    /// Convinience method that gets [Stream] if it's found on the server or creates it with
```

Comment:
Lets just say what it does, rather than talk about convenience, utility, etc?

```suggestion
    /// Convinience method that gets [Stream] if it's found on the server or creates it with
```

Reply:
I majority of the API - agree. But hm, it his case, I think it's good to talk about use case and explain if given function is just a utility that does not introduce any new/different behaviour? Just to make things clear?
We can move that sentence at the end of the doc comment if that would be a good compromise for you(?).

Reply:
Would write something like: 

```
/// Creates a stream with the given configuration on the server if it is not present, then returns a handle to the stream  on the server.
///
/// Note: This does not validate if the Stream on the server is compatible with the configuration passed in.
```

The first line is what you're gonna see in editors.